101-documentdb-account-create-multi-region-account/azuredeploy.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "databaseAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "The DocumentDB database account name."
+      }
+    },
+    "writeRegionLocation": {
+      "type": "string",
```

Comment:
Thinking about this a bit more, I think this is debatable either way; personally I'd prefer to have the default values for the regions in the "defaultValue" of the azuredeploy.json file for the reasons described in that wiki page, but I'll leave the decision up to you which you prefer :)